bbc-newslabs-footer.js

```diff
@@ -161,6 +161,12 @@ xml:space="preserve">
                     'mailto:newslabs-development@lists.forge.bbc.co.uk?subject='
                     + encodeURI(newvalue + " feedback")
             }
+            else if (name=='Transcriptor')
```

Comment:
where does `name` get set to Transcriptor?

Reply:
If you view source of Transcriptor you'll see:

```
<bbc-newslabs-header beta app="Transcriptor" subtitle="Transcription Service"></bbc-newslabs-header>
```

Reply:
great - thanks @bennuttall!

Reply:
Yes, i went to "View Page Source" in the Transcriptor UI and got that name from there.